gpt_engineer/file_selector.py

```diff
@@ -187,7 +188,8 @@ def ask_for_selection(self) -> List[str]:
         elif re.match(regex, user_input):
             try:
                 user_input = (
-                    user_input.replace("", ",") if " " in user_input else user_input
+                    user_input.replace(
+                        "", ",") if " " in user_input else user_input
```

Comment:
Not sure what this is for but shouldn't it be this:

```suggestion
                        " ", ",") if " " in user_input else user_input
```

Reply:
Fixed.